app/controllers/cards_controller.rb

```diff
@@ -1,5 +1,50 @@
 class CardsController < ApplicationController
-    def index
-        @cards = Card.all
+  before_filter :find_card,
+      :only => [:show, :update, :edit, :destroy]
+
+  def index
+    @cards = Card.all
+  end
+
+  def create
+    @card = Card.new(card_params)
+    @card.review_date = DateTime.now.next_day(3)
```

Comment:
Do not use `DateTime.now` without zone. Use one of `Time.zone.now`, `DateTime.now.current`, `DateTime.now.in_time_zone`, `DateTime.now.utc`, `DateTime.now.getlocal`, `DateTime.now.iso8601`, `DateTime.now.jisx0301`, `DateTime.now.rfc3339`, `DateTime.now.to_i`, `DateTime.now.to_f` instead.